src/Stratis.Bitcoin.Features.Consensus.Tests/InitialBlockDownloadTest.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Base;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests
+{
+    public class InitialBlockDownloadTest
+    {
+        private readonly ConsensusSettings consensusSettings;
+        private readonly Checkpoints checkpoints;
+        private readonly NodeSettings nodeSettings;
+        private readonly ChainState chainState;
+
+        public InitialBlockDownloadTest()
+        {
+            this.consensusSettings = new ConsensusSettings(NodeSettings.Default(), new LoggerFactory());
```

Comment:
Change this ExtendedLoggerFcatory so that its more consistent with what the rest of the code base.

Reply:
Fixed